src/main.cpp

```diff
@@ -504,9 +533,6 @@ void influxDbAddTags(char* tags) {
 }
 
 bool influxDbWrite() {
-  if(apm25 == 0 || apm10 == 0) {
```

Comment:
Please sync your branch with the current devel branch, because this lines already committed on previous PR #64 fix

Reply:
donde